src/network/rpc/Test.cpp

```diff
@@ -102,26 +98,18 @@ Json::Value Test::create_test_coin_transactions(const Json::Value &param1) {
       if (!param1["secret"].empty() && !param1["secret"].asString().empty()) {
         sk = secret_t(param1["secret"].asString());
       }
-      if (trx_creater_.wait_for(std::chrono::seconds(0)) != std::future_status::ready) {
-        res = "Busy in creating transactions ... please try later ...";
-      } else {
-        trx_creater_ = std::async(std::launch::async, [node, &log_time, delay, number, nonce, receiver, sk]() {
-          // get trx receiving time stamp
-          bytes data;
-          uint i = 0;
-          while (i < number) {
-            auto now = getCurrentTimeMilliSeconds();
-            val_t value = val_t(100);
-            auto trx = taraxa::Transaction(i + nonce, value, 1000, 0, data, sk, receiver);
-            LOG(log_time) << "Transaction " << trx.getHash() << " received at: " << now;
-            node->getTransactionManager()->insertTransaction(trx, false, true);
-            thisThreadSleepForMicroSeconds(delay);
-            i++;
-          }
-        });
-
-        res = "Creating " + std::to_string(number) + " transactions ...";
+      // get trx receiving time stamp
+      uint i = 0;
+      while (i < number) {
+        auto now = getCurrentTimeMilliSeconds();
+        val_t value = val_t(100);
+        auto trx = taraxa::Transaction(i + nonce, value, 1000, 0, bytes(), sk, receiver);
+        LOG(log_time) << "Transaction " << trx.getHash() << " received at: " << now;
+        node->getTransactionManager()->insertTransaction(trx, false, true);
+        thisThreadSleepForMicroSeconds(delay);
+        i++;
```

Comment:
you are changing the same stuff as in https://github.com/Taraxa-project/taraxa-node/pull/1046/files. Please clean up

Reply:
Will be rebase after merging the PR and this change will gone

Reply:
but why would you do it like this ? Those PR's are independent arent they ?

Reply:
Will remove this commit from there. There is no sufficient reason for it 🥴